tests/testMeasureSources.py

```diff
@@ -34,6 +34,7 @@
 import numpy as np
 
 import lsst.pex.logging as pexLogging
+from lsst.log import Log
 import lsst.pex.exceptions
```

Comment:
Did you forget to remove this, and the code below that uses it? (I didn't see any other instances, but suggest a global search to be sure).